controllers/chat/initChatFromProfileAsAnonymousV2.js

```diff
@@ -152,8 +150,10 @@ const initChatFromProfileAsAnonymousV2 = async (req, res) => {
 
     return res.status(200).json(responseSuccess('sent', response));
   } catch (error) {
-    await client.disconnectUser();
+    console.log('Error', error);
     return ErrorResponse.e400(res, error.message);
+  } finally {
+    await client.disconnectUser();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While moving the `client.disconnectUser()` to a `finally` block is a good practice to ensure it always runs, logging errors directly to the console can be considered a bad practice. It would be better to use a dedicated logging service that can provide more context and handle different log levels. This way, you can filter out unnecessary logs in production and only log errors or important information.

```diff
-    console.log('Error', error);
+    logger.error('Error', error);
     return ErrorResponse.e400(res, error.message);
```
Please replace `console.log` with a proper logging mechanism like `winston`, `bunyan`, or any other preferred logging library.


<!-- This is an auto-generated comment by OSS CodeRabbit -->